CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java

```diff
@@ -761,7 +761,7 @@ private void partition(final int recursiveCount, final boolean fast, final boole
         if (verbose) {
             if (config.getVerbosity() == 0) {
                 final String fmt = String.format("%s.%df", "%", config.getVerboseNumberPrecision());
-                System.out.printf("to %d modules with codelength " + fmt + "%n", getNumTopModules(), codelength);
+                System.out.printf("to %d modules with codelength %s%n", getNumTopModules(), fmt, codelength);
```

Comment:
This is the one I'm unsure about. Not sure whether I actually fixed this properly. `codelength` threw me out

Reply:
I'd agree. It looks like the wrong number of args.